observe/observe.go

```diff
@@ -0,0 +1,110 @@
+// Package observe provides functions
+// that help with setting tracing/metrics
+// in cloud providers, mainly GCP.
+package observe
+
+import (
+	"context"
+	"os"
+
+	traceapi "cloud.google.com/go/trace/apiv2"
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"contrib.go.opencensus.io/exporter/stackdriver/monitoredresource"
+	"go.opencensus.io/exporter/prometheus"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+// NewStackDriverExporter will return the tracing and metrics through
+// the stack driver exporter, if exists in the underlying platform.
+// If exporter is registered, it returns the exporter so you can register
+// it and ensure to call Flush on termination.
+func NewStackDriverExporter(projectID string, onErr func(error)) (*stackdriver.Exporter, error) {
```

Comment:
From their product page, the "D" in Stackdriver is lowercase: https://cloud.google.com/stackdriver/

Mind tweaking that through to keep "on brand"?

This should be the last request, promise!

Reply:
@jprobinson no worries, updated